packaging/rpm/microshift.spec

```diff
@@ -61,7 +61,16 @@ Requires: sos
 %{?systemd_requires}
 
 %description
-The MicroShift package provides an OpenShift Kubernetes distribution optimized for small form factor and edge computing.
+The microshift package provides an OpenShift Kubernetes distribution optimized for small form factor and edge computing.
+
+
+%package release-info
+Summary: Release information for MicroShift
+BuildArch: noarch
+
+%description release-info
+The microshift-release package provides release information files for this release.
```

Comment:
May be we should explain the purpose of this; i.e.

This package can be used to find the images consumed by MicroShift in runtime
and embed those into blueprints/ostree images.

Reply:
Added.